themes/bootstrap3/js/common.js

```diff
@@ -479,7 +479,43 @@ function setupQRCodeLinks(_container) {
   });
 }
 
+/**
+ * Function to check if browser supports current js.
+ */
+function checkForBrowserSupport() {
+  /* jshint ignore:start */
+  /* eslint-disable */
+  try {
+    // Arrow functions support
+    () => { };
+    // Class support
+    class __ES6FeatureDetectionTest { };
+    // Object initializer property and method shorthands
+    let a = true;
+    let b = { 
+      a,
+      c() { return true; },
+      d: [1, 2, 3],
+    };
+    // Object destructuring
+    let {c, d} = b;
+    // Spread operator
+    let e = [...d, 4, c];
+    if (e.length !== 5) {
+      throw new Error('Unexpected result');
+    }
+  } catch (exception) {
+    var outdated = document.getElementById('browser-outdated');
+    if (outdated) {
+      outdated.classList.remove('hidden');
+    }
+  }
+  /* eslint-enable */
+  /* jshint ignore:end */
+}
+
 $(document).ready(function commonDocReady() {
+  checkForBrowserSupport();
```

Comment:
I tried this out in [Firefox 43](https://filehippo.com/download_mozilla-firefox-64/43.0.4/) ([two versions before ES6 support](https://caniuse.com/es6-class)) and this check didn't parse, which broke all of `common.js` and led to the VuFind object not existing.

![image](https://user-images.githubusercontent.com/451107/157063356-e91e1944-ae91-4a4f-98c2-3eab3e3471f2.png)

Reply:
I actually had to change a few `let variable`s to `var variable`s to get this far.

Reply:
Going to test it with the same browser. But thats odd.